ubports/ubuntu-touch
# Change to README.md

### Requirements

[✓] Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
[✓] All new code requires tests to ensure against regressions

### Description of the Change

I have been a markdown user for a couple years as of now. What I noticed from the original README.md file was that sections were a tiny bit too squished together and I believed I could change it to look more nice. (I'm hoping to add more changes to further projects you all have made).

Here is what I did:
* Added a couple small icons near the title.
* Added lines between sections to make it seem more neatly organized.
* Spaced out sections a tiny bit more from the **Title**,  "How to use this repo", "Shortcut to issue milestones", and "Labels and Filtering".

### Benefits



Benefits from this addition I had made is that it will allow the page to be more readable and seem more neat. It increases user-friendliness with the use of lines and separation between parts of the README. With a neat and well-setup README, users can understand more about a project and not overlook some critical details.

### Possible Drawbacks



The only possible side-effect I could find is that maybe some users may not like the use of items such as emojis, icons, or tiny images. However, this is very unlikely.

### QA-Steps



You can look at the README.md file and see if you like the changes. <br>
If you really want to test it, grab a nearby person and ask if they like the version of this README.md file I made or the old README.md file. <br>
Do this with 1-5 randomly selected people *(as to not have a bias and to allow for the use of random selection)* and use the score to see if this is a better change.

### Related Issues


*N/A*
